Wangscape/CMakeLists.txt

```diff
@@ -1,17 +1,26 @@
 set(lib-src
     MetaOutput.cpp
     Options.cpp
+    OptionsManager.cpp
```

Comment:
Yes, there's a thing called `GLOB`, which allows to include files that conform to the given pattern. I thought that both ways to do it are equally good, but I've found [this answer](http://stackoverflow.com/a/32412044/4169253) which convice me to list files explicitly.

Now we've got one library built from source files, when we got the second, with such list it will be easier to say, which files every library is built from.